roles/matrix-synapse/files/workers-doc-to-yaml.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+# Fetch the synapse worker documentation and extract endpoint URLs
+# matrix-org/synapse master branch points to current stable release
+
+URL=https://github.com/matrix-org/synapse/raw/master/docs/workers.md
+curl -L ${URL} | awk -f workers-doc-to-yaml.awk > ../vars/workers.yml
```

Comment:
Is this still needed after `vars/workers.yml` was added?

Reply:
I guess this shell script is just some self-executing documentation telling developers how to trigger the regeneration of `vars/workers.yml`. This is also documented in `workers-doc-to-yaml.awk`.

Reply:
Yes, but then it should maybe be moved to `contrib` or similar, if it is not used in the playbook, as leaving it at this place implies it is a used component which needs maintenance.